test/services/pscService.unit.ts

```diff
@@ -5,7 +5,7 @@ import { HttpStatusCode } from "axios";
 import { Request } from "express";
 import { createOAuthApiClient } from "../../src/services/apiClientService";
 import { getPscIndividual } from "../../src/services/pscService";
-import { COMPANY_NUMBER, PSC_ID, PSC_INDIVIDUAL } from "../mocks/psc.mock";
+import { COMPANY_NUMBER, PSC_ID, PSC_INDIVIDUAL } from "../mocks/Psc.mock";
```

Comment:
File was renamed to "psc.mock.ts" by an earlier PR, so this gives me missing import error.